woocommerce/woocommerce
# Add psuedo random option for remote inbox note A/B tests

### All Submissions:






### Changes proposed in this Pull Request:

An option `woocommerce_inbox_variant_assignment` is added on install or update to supply a value to be used by Remote Inbox Notifications rules to determine which variant of a particular Inbox Note get delivered to a merchant.

### How to test the changes in this Pull Request:

1. Ensure `woocommerce_inbox_variant_assignment` option does not exist. If it does, delete it before testing.
2. Deactivate the plugin then activate it again
3. `woocommerce_inbox_variant_assignment` option should be created.
4. Repeat step 2 and ensure that the value of `woocommerce_inbox_variant_assignment` does not change.

### Other information:




### Changelog entry

- dev: Introduce an option for assignment of variations for Remote Inbox Notification A/B testing
